runtime/moonriver/src/lib.rs

```diff
@@ -407,7 +407,7 @@ parameter_types! {
 	/// )
 	pub const GasLimitPovSizeRatio: u64 = 4;
 	/// The amount of gas per storage (in bytes).
-	pub GasLimitStorageGrowthRatio: u64 = 0;
+	pub GasLimitStorageGrowthRatio: u64 = 366;
```

Comment:
Can you make it a formula based on the total gas avaiable and a variable which would be the maximum bytes per block ?

Reply:
It appears that this expression is evaluated on block execution. so it's better to use a constant (https://github.com/moonbeam-foundation/moonbeam/pull/2452#discussion_r1309400272). I've added rust test to verify that the constant matches the formula.

Reply:
Can't you make a parameter that is the formula and assign the parameter to this pub ?

Reply:
I couldn't find a way to accomplish this. The formula for calculating `GasLimitStorageGrowthRatio` relies on `BlockGasLimit`, which is determined by a non-constant formula. Therefore, it is not possible to define a constant parameter based on the formula outside of `parameter_types!` and assign it to `GasLimitStorageRatio`.

Reply:
I don't really understand.... Those are constants in the logic of the runtime, why don't we move them into a const at compilation time and simply have the pub return them (BlockGasLimit is also constant) ?